examples/typescript/README.md

```diff
@@ -0,0 +1,44 @@
+# saucectl puppeteer & typescript example
+
+Example running saucectl with puppeteer & typescript.
+
+## What You'll Need
+
+The steps below illustrate one of the quickest ways to get set up. If you'd like a more in-depth guide, please check out
+our [documentation](https://docs.saucelabs.com/testrunner-toolkit/installation).
+
+### Install `saucectl`
+
+```shell
+npm install -g saucectl
+```
+
+### Set Your Sauce Labs Credentials
+
+```shell
+saucectl configure
+```
+
+## Ensure you already have typescript installed
+
+```shell
+npm install -g typescript
+```
+
+## Run the TypeScript compiler like so
+
+Simply check out this repo, navigate into `examples/typescript` and run the command below:
+
+```shell
+tsc -p ./tsconfig.json
```

Comment:
I think we need a step before this one to `npm install`? Some compile time dependencies are missing when I follow these steps.

Reply:
Actually I'm still getting a compile error after `npm install`.

```
🍖 npx tsc -p ./tsconfig.json
error TS2688: Cannot find type definition file for 'mocha'.
  The file is in the program because:
    Entry point of type library 'mocha' specified in compilerOptions

  tsconfig.json:7:15
    7     "types": ["mocha", "jest"],
                    ~~~~~~~
    File is entry point of type library specified here.


Found 1 error.
```

Reply:
I guess I missed adding `npm install` because `package.json` has `mocha` and `jest` dependencies.

Reply:
> I think we need a step before this one to `npm install`? Some compile time dependencies are missing when I follow these steps.

I added this step.